t/tasks/inventory/macos/hostname.t

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use lib 't/lib';
+
+use Test::Deep;
+use Test::Exception;
+use Test::More;
+use Test::NoWarnings;
+
+use FusionInventory::Agent::Task::Inventory::MacOS::Hostname;
+
+my %tests = (
+    'hostname' => 'mac-fusion01'
```

Comment:
Yeah, you're right, but for the moment, as I had just one sample, I just created one test. But sure, no problem, biloute !